src/stactools/naip/stac.py

```diff
@@ -121,30 +124,51 @@ def create_item(
         )
 
     if fgdc_metadata_href is not None:
-        fgdc_metadata_text = read_text(fgdc_metadata_href)
-        fgdc = parse_fgdc_metadata(fgdc_metadata_text)
-    else:
-        fgdc = {}
+        if year == "2020":
```

Comment:
Is it _only_ 2020 that uses XML? Or will they be using XML going forward (i.e. should this be `if int(year) >= 2020`)?

Reply:
it's unknown right now -- 2021 isn't out yet.

I would prefer we explicitly fail if the year isn't one of the ones we know we support, rather than have some undefined behavior when 2021+ does come out.